examples/add/package.json

```diff
@@ -1,9 +1,10 @@
 {
   "scripts": {
-    "build": "webpack",
-    "serve": "webpack-dev-server"
+    "build": "webpack -p",
+    "serve": "webpack-dev-server -p"
   },
   "devDependencies": {
+    "@wasm-tool/wasm-pack-plugin": "0.2.0",
```

Comment:
I published `@wasm-tool/wasm-pack-plugin@0.2.1` in the meantime, could you please update it? or maybe the bot will update it after?